src/routes/quickstarts/cypress/setup.mdx

```diff
@@ -0,0 +1,180 @@
+import { Tabs, Tab } from "../../../components/tabs"
+import Snippet from "../../../components/snippet"
+
+# Mock Network Requests in Cypress with Mirage
+
+Use your Mirage server to test your application under different server scenarios using [Cypress](https://www.cypress.io/).
+
+> This is a quickstart guide for people already using `cypress` in their apps.
+
+## Step 1: Install Mirage
+
+First, make sure you have Mirage installed:
+
+```bash
+# Using npm
+npm install --save-dev miragejs
+
+# Using Yarn
+yarn add --dev miragejs
+```
+
+## Step 2: Define your server
+
+Create a new `src/server.js` file and define your mock server.
+
+Here's a basic example:
+
+```js
+// src/server.js
+import { Server, Model } from "miragejs"
+
+export function makeServer({ environment = "development" } = {}) {
+  let server = new Server({
+    environment,
+
+    models: {
+      user: Model,
+    },
+
+    seeds(server) {
+      server.create("user", { name: "Bob" })
+      server.create("user", { name: "Alice" })
+    },
+
+    routes() {
+      this.namespace = "api"
+
+      this.get("/users", schema => {
+        return schema.users.all()
+      })
+    },
+  })
+
+  return server
+}
+```
+
+## Step 3: Add the Mirage Cypress configuration
+
+Add the following Mirage bootstrapping code and command to your `cypress/support/index.js` file:
```

Comment:
Let's fill this out a little (also I don't think we're adding a command anymore). Let's say

Add the following code to your `cypress/support/index.js` file:

// code

This code defines a `handleFromCypress` function on your application's `window` object. In the next step, we'll configure your app to call this function whenever it makes a network request and Cypress is running.